app/_hub/kong-inc/jwt-signer/overview/manage-keys.md

```diff
@@ -0,0 +1,59 @@
+---
+nav_title: Manage key signing
+---
+
+What is key signing and why does the JWT plugin require it?
+
+With the JWT signer plugin, you have the option to either use signing keys that are externally managed by you or to use Kong-generated signing keys. If you're using the JWT signer plugin with {{site.konnect_short_name}}, you *must* manage your signing keys externally. If you're using {{site.base_gateway}}, you can either manage your signing keys externally or use Kong-generated keys.
+
+Are there any reasons why someone who is using {{site.base_gateway}} might choose one mode over the other?
+
+## Externally manage signing keys
+
+This section provides instructions that explain how to host key sets using the library of your choice and configure the JWT plugin to load your externally managed keys.
+
+### Prerequisites
+
+* A keys host (for example, a Docker container) for the keys. This host must be accessible from both {{site.base_gateway}} and upstreams with NGINX installed on it
+* An environment where you can generate signing keys
+What does this mean? What do we need in the environment?
+* A TLS certificate to secure the key server
+
+### Host key sets
+
+1. Generate server certificates using your organization's certificate authority (CA).
+
+1. Generate JWKs using any of the [standard libraries](https://jwt.io/libraries) that align with your organization's security requriements. 
+    The key size (the modulo) for RSA keys is currently hard-coded to 2048 bits.
+
+1. Copy the generated keys in a directory on the host machine. Also, copy your server certificate and key to the host machine as well. 
+
+1. Configure your host server with the certificate, certificate key, signing keys, and JSON Web Key Set (JWKS). 
+    
+    {:.note}
+    > **Note:** External URLs that contain private keys should be protected so that only Kong can access them. Currently, Kong doesn't add any authentication headers when it loads the keys from an external endpoint, so you have to do it with network level restrictions. If you commonly need to manage private keys externally instead of allowing Kong to autogenerate them, you can add another parameter for adding an authentication header (possibly similar to `config.channel_token_introspection_authorization`).
+
+### Configure the JWT plugin with your self-hosted key sets
+
+To configure the JWT plugin with your self-hosted key sets, you must specify `config.access_token_keyset` or `config.channel_token_keyset` with either an `http://` or `https://` prefix:
+
+```bash
+curl -X POST \
```

Comment:
@lena-larionova I also thought that maybe https://docs.konghq.com/hub/kong-inc/jwt-signer/how-to/basic-example/ was missing a Konnect example? I pulled this content from another file with a Konnect API tab, so I was wondering if I should/if there was a way to get this added to the basic example config (since all Konnect users will need this). I looked at the plugin toolkit and wasn't sure where to add to the examples.

Reply:
This plugin isn't supported in Konnect, it incorrectly had a konnect badge (already fixed on the site). So that part is right.